packages/web-client/app/utils/web3-strategies/ethereum.ts

```diff
@@ -1,7 +1,8 @@
 import Layer1ChainWeb3Strategy from './layer1-chain';
+import { networkDisplayInfo } from './network-display-info';
 
 export default class EthereumWeb3Strategy extends Layer1ChainWeb3Strategy {
   constructor() {
-    super('mainnet', 'Ethereum mainnet');
+    super('mainnet', networkDisplayInfo.mainnet.fullName);
```

Comment:
Now that there’s a `networkDisplayInfo` to collect these programmatic and UI names, maybe this second argument to the constructor is unnecessary and can be derived in the constructors for the strategies? 🤔

Reply:
This seems good, it reduces the number of files that need to know about `networkDisplayInfo`.

Reply:
Will change